v2/src/core/router/index.ts

```diff
@@ -95,7 +95,7 @@ const routes: Array<RouteRecordRaw> = [
                 name: ROUTE_NAME.ADDRESS_TOKENS.NAME,
                 component: AddressTokensView,
                 props: route => ({ tab: route.query.t }),
-                beforeEnter: addressTokensRouteGuard
+                beforeEnter: addressRouteGuard('tokens')
```

Comment:
please use the ADDRESS_ROUTE_QUERY variable instead of the string or make a new enum